mmv1/third_party/terraform/tests/resource_compute_disk_async_replication_test.go

```diff
@@ -0,0 +1,140 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeDiskAsyncReplication(t *testing.T) {
+	t.Parallel()
+
+	region := GetTestRegionFromEnv()
+	if !stringInSlice([]string{"europe-west2", "europe-west1", "us-central1", "us-east1", "us-west1", "us-east4", "asia-east1", "australia-southeast1"}, region) {
+		return
+	}
+	secondaryRegion := region
+	switch region {
+	case "europe-west2":
+		secondaryRegion = "europe-west1"
+	case "europe-west1":
+		secondaryRegion = "europe-west2"
+	case "us-central1":
+		secondaryRegion = "us-east1"
+	case "us-east1", "us-west1", "us-east4":
+		secondaryRegion = "us-central1"
+	case "asia-east1":
+		secondaryRegion = "asia-southeast1"
+	case "asia-southeast1":
+		secondaryRegion = "asia-east1"
+	case "australia-southeast1":
+		secondaryRegion = "australia-southeast2"
+	case "australia-southeast2":
+		secondaryRegion = "australia-southeast1"
+	}
+
+	primaryDisk := fmt.Sprintf("tf-test-disk-primary-%s", RandString(t, 10))
+	secondaryDisk := fmt.Sprintf("tf-test-disk-secondary-%s", RandString(t, 10))
+	primaryRegionalDisk := fmt.Sprintf("tf-test-disk-rprimary-%s", RandString(t, 10))
+	secondaryRegionalDisk := fmt.Sprintf("tf-test-disk-rsecondary-%s", RandString(t, 10))
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderBetaFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeDiskAsyncReplication_basicZonal(region, secondaryRegion, primaryDisk, secondaryDisk),
+			},
```

Comment:
Could we add sequential steps where we `ImportStateVerify`.. This would confirm that the imported state is identical to the provisioned state.